packages/lab/src/list-next/ListItemNext.css

```diff
@@ -40,23 +40,42 @@
   outline-offset: calc(var(--salt-focused-outlineWidth) * -1);
 }
 
+/* active list item on mouse hover and keyboard focus */
 .saltListItemNext:hover:not([role="presentation"]):not(.saltListItemNext-disabled),
 /* TODO: do we need to add the role exception to the below classes? */
 .saltListItemNext-focused {
   --list-item-background: var(--salt-selectable-background-hover);
   cursor: var(--salt-selectable-cursor-hover);
 }
 
-.saltListItemNext[aria-selected="true"] {
-  border-block: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-selectable-borderColor-selected);
+/* active list item when selected */
+.saltListItemNext[aria-selected="true"]:not(.saltListItemNext-disabled) {
```

Comment:
we're trying to avoid using 'not', could this be split out into an override if also disabled?